kolibri/core/webpack/hooks.py

```diff
@@ -152,13 +152,23 @@ def webpack_bundle_data(self):
                 "events": self.events,
                 "once": self.once,
                 "static_url_root": getattr(django_settings, 'STATIC_URL'),
-                "locale_data_folder": os.path.join(getattr(django_settings, 'LOCALE_PATHS')[0], 'en', 'LC_FRONTEND_MESSAGES'),
+                "locale_data_folder": self.locale_data_folder,
                 "version": self.version,
             }
         else:
             logger.warn("{src_file} not found for plugin {name}.".format(src_file=self.src_file, name=self.unique_slug))
 
     @property
+    def locale_data_folder(self):
```

Comment:
I was wondering if the property name was too general, but in a sense - even though it is configurable - it's supposed to be used as a convention.